flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -301,18 +301,6 @@ public Builder uidPrefix(String newPrefix) {
       return appendDummySink(committerStream);
     }
 
-    /**
-     * Append the iceberg sink operators to write records to iceberg table.
-     *
-     * @return {@link DataStreamSink} for sink.
-     * @deprecated this will be removed in 0.14.0; use {@link #append()} because its returned {@link DataStreamSink}
-     * has a more correct data type.
-     */
-    @Deprecated
-    public DataStreamSink<RowData> build() {
-      return chainIcebergOperators();
-    }
-
```

Comment:
We can not remove this `build` method in current version, in case of breaking the downstream flink application !  As the javadoc says,  0.14.0 is the correct release to remove this.

Reply:
Got it. and modify PR title